### PR TITLE
[master] build: prepare v0.15.1.0 release

### PR DESCRIPTION
**Pull request by selsta**
Will create a separate PR for branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -219,7 +219,7 @@ invokes cmake commands as needed.
 
     ```bash
     cd monero
-    git checkout release-v0.15
+    git checkout release-v0.15.1
```
Comment (moneromooo-monero) [inline README.md:222]: Is that intended ? Up to now it was a x.y branch, further numbers were tags.

File: src/checkpoints/checkpoints.cpp [hunk 2]
```diff
@@ -211,6 +211,8 @@ namespace cryptonote
     ADD_CHECKPOINT(1775600, "1c6e01c661dc22cab939e79ec6a5272190624ce8356d2f7b958e4f9a57fdb05e");
     ADD_CHECKPOINT(1856000, "9b57f17f29c71a3acd8a7904b93c41fa6eb8d2b7c73936ce4f1702d14880ba29");
     ADD_CHECKPOINT(1958000, "98a5d6e51afdf3146e0eefb10a66e8648d8d4d5c2742be8835e976ba217c9bb2");
+    ADD_CHECKPOINT(2046000, "5e867f0b8baefed9244a681df97fc885d8ab36c3dfcd24c7a3abf3b8ac8b8314");
+    ADD_CHECKPOINT(2092500, "c4e00820c9c7989b49153d5e90ae095a18a11d990e82fcc3be54e6ed785472b5");
```
Comment (moneromooo-monero) [inline src/checkpoints/checkpoints.cpp:215]: That's like 60 blocks ago. *Very* close.

File: README.md (hunk 1 above)
Comment (selsta) [reply README.md:222]: I thought we will branch `release-v0.15.1` from master. Would you prefer merging master into `release-v0.15`?

File: src/checkpoints/checkpoints.cpp (hunk 2 above)
Comment (selsta) [reply src/checkpoints/checkpoints.cpp:215]: Plan is to leave this PR open until shortly before tagging in 4-5 days. Should be ok then?

Comment (moneromooo-monero) [reply src/checkpoints/checkpoints.cpp:215]: Sounds ok.

File: README.md (hunk 1 above)
Comment (moneromooo-monero) [reply README.md:222]: No. I'd have expected 0.16 then. But then it doesn't really matter, fine that way.

**Pull request closed without merging**